_saas-integrations/impact/v1/impact-v1.md

```diff
@@ -0,0 +1,131 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Impact
+permalink: /integrations/saas/impact
+keywords: impact, integration, schema, etl impact, impact etl, impact schema
+layout: singer
+# input: false
+
+key: "impact-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "impact"
+display_name: "Impact"
+
+singer: true
+status-url: ""
+
+tap-name: "Impact"
+repo-url: https://github.com/singer-io/tap-impact
+
+this-version: "1"
+
+api: |
+  [Impact REST API](https://developer.impact.com/default){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true
+
+api-type: "platform.impact"
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+requirements-info: |
+  There are no role types in {{ integration.display_name }}, but you do need to have API Access enabled on your account to access the API info. To do this, login to the {{ integration.display_name }} console. Locate the gear icon. Then, go to **Technical Settings > API Access**. There you can click **Enable API Access Now**.
```

Comment:
1. This section isn't showing up on the base doc. The layout the page uses (`_layouts/singer.html`) looks for the presence of `requirements-list` to create the **Prerequisites** section - this alone won't do it. I suggest using `requirements-list` like you normally would:

   ```yaml
   requirements-list:
     - item: |
         [content]
   ```
2. Why mention role types if Impact doesn't have them?